toxav/rtp.c

```diff
@@ -343,27 +337,20 @@ static void update_bwc_values(const Logger *log, RTPSession *session, const stru
  *   payload.
  * @param incoming_data_length The packet length *not* including header, i.e.
  *   this is the actual payload length.
- * @param log A logger.
```

Comment:
There is still `log` in the function parameters, it's now the 1st parameter, yet the comment for it is now gone entirely.

Reply:
added